content/doc/book/system-administration/monitoring.adoc

```diff
@@ -69,7 +69,7 @@ https://issues.jenkins.io/browse/JENKINS-25695[JENKINS-25695].
 Sometimes, for example https://wiki.jenkins.io/display/JENKINS/Remoting+issue[to diagnose the agent connection loss
 problem], you may want to disable the ping thread.  This needs to be done in two places.
 
-To disable the controller from pinging agents, the system property `+hudson.slaves.ChannelPinger.pingInterval+` should
+To disable the controller from pinging agents, the system property `+hudson.slaves.ChannelPinger.pingIntervalSec+` should
```

Comment:
Needs to use the correct symbol.  See the [Jenkins core source code](https://github.com/jenkinsci/jenkins/blob/jenkins-2.393/core/src/main/java/hudson/slaves/ChannelPinger.java#L62) for the symbol
```suggestion
To disable the controller from pinging agents, the system property `+hudson.slaves.ChannelPinger.pingIntervalSeconds+` should
```